extensions/cornerstone/src/init.js

```diff
@@ -32,9 +33,10 @@ cornerstone.metaData.addProvider(fallbackMetaDataProvider, -1);
  * @param {Object|Array} configuration.csToolsConfig
  */
 export default function init({ servicesManager, configuration }) {
```

Comment:
Just to make init cleaner, we should shift these to two private methods:

```
_subscribeToMeasurementService()
_listenAndSyncCornerstoneMeasurements()
```

Or something similar, but with better names

Reply:
Updated, thanks.